controller/cmd/destination/main.go

```diff
@@ -56,7 +58,11 @@ func Main(args []string) {
 	go func() {
 		log.Infof("starting admin server on %s", *metricsAddr)
 		if err := adminServer.ListenAndServe(); err != nil {
-			log.Errorf("failed to start destination admin server: %s", err)
+			if errors.Is(err, http.ErrServerClosed) {
+				log.Infof("shutting down admin server on %s", *metricsAddr)
+			} else {
+				log.Errorf("failed to start destination admin server: %s", err)
```

Comment:
```suggestion
			if errors.Is(err, http.ErrServerClosed) {
				log.Infof("Admin server closed (%s)", *metricsAddr)
			} else {
				log.Errorf("Admin server error (%s): %w", *metricsAddr, err)
```

Some minor nits: %w is necessary to prevent log injection.